### ISSUE TITLE
Bug - a11y violation - duplicate id when same item in Top Stories and Features & Analysis

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#9848]
A few live E2Es failing with a11y violations because of the same article in Top Stories and Features & Analysis being present on news amp. This will always happen if the features and analysis article becomes a top story. In this case it has the same aria-labelledby id ‘promo-newsuk60264106-feature-1’
Would it be possible for the id to be slightly different depending on which section it is placed in? With an extra part like ‘top-stories’ or ‘features-analysis’? 

<img width="519" alt="Screenshot 2022-02-09 at 10 54 39" src="https://user-images.githubusercontent.com/54575620/153204769-8a1eebc7-f43e-4262-b205-8dc1adb6973e.png">

URLs: https://www.bbc.com/news/uk-56342465.amp
https://www.bbc.co.uk/news/articles/cj7xrxz0e8zo.amp
